pennylane/io.py

```diff
@@ -407,6 +407,61 @@ def from_qiskit_op(qiskit_op, params=None, wires=None):
         raise RuntimeError(_MISSING_QISKIT_PLUGIN_MESSAGE) from e
 
 
+def from_qiskit_noise(noise_model, **kwargs):
+    """Converts a Qiskit `NoiseModel <https://qiskit.github.io/qiskit-aer/stubs/qiskit_aer.noise.NoiseModel.html>`__
+    into a PennyLane :class:`~.NoiseModel`.
+
+    Args:
+        noise_model (qiskit_aer.noise.NoiseModel): a Qiskit ``NoiseModel`` instance.
+        kwargs: Optional keyword arguments for conversion of the noise model.
+
+    Keyword Arguments:
+        verbose (bool): show a complete list of Kraus matrices for ``qml.QubitChannel`` instead of
+            the number of Kraus matrices and the number of qubits they act on. The default is ``False``.
```

Comment:
```suggestion
        verbose (bool): when printing a ``NoiseModel``, a complete list of Kraus matrices for each 
        ``qml.QubitChannel`` can be displayed (``verbose = True``). By default, ``verbose=False`` 
        and only the number of Kraus matrices and the number of qubits they act on is displayed 
        for brevity. 
```

@trbromley what do you think?